index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Styles.css">
```

Comment:
Your css file is named Style.css with an uppercase S but the file you are referring to here is style.css with a lower case s. Please link the file as below

    <link rel="stylesheet" href="Styles.css">

Rename either Style.css file or Styles.css link to make it a relative path and so it will work on any deployment environment 👍🏾

Happy coding

Reply:
- [ ] Good attempt @maserekafelex , this comment is confusing and not clear. Line 7 clearly shows that the css file linked (reffered to)  is name Style.css but you clain its named style.css. ⚔️  Please be clear about this .

Reply:
The css file linked is named Styles.css which is a plural while the file css (to be rendered to) is single i.e Style.css